### DISCUSSION TITLE
How to generate '/' Index-route for 'Index.vue' instead of 'index.vue' ?

### DISCUSSION THREAD

**Discussion by Wolverin-e** [posva/unplugin-vue-router#174] [Questions and Help] [answered]
I see that plugin automatically generates index-routes for "index.vue" files.
How to enable similar behaviour the same be done for "Index.vue" files?

**Comment by posva** [comment, marked as answer]
It's currently not possible but I really discourage you from using Upper case in files as it creates very difficult-to-debug problems when other people's machines, so I think it's better to enforce this convention

**Reply by Wolverin-e** [reply]
Hi, Thanks for the quick answer. I think it resolves my question for now.
Also, It would be great if you could share some reference/links to known-issues with UpperCase naming.

**Reply by posva** [reply]
It’s from real world experiences 😄 I don’t have links

**Reply by Wolverin-e** [reply]
Oh, okay. 👍

**Reply by stafyniaksacha** [reply]
Windows does not make difference between lowercase and uppercase files names, whereas linux does
